build.gradle

```diff
@@ -133,17 +141,28 @@ dependencies {
     // dependency only for obtaining default configuration
     compile 'io.gitlab.arturbosch.detekt:detekt-cli:1.0.0.RC3'
 
+    compileOnly 'org.projectlombok:lombok:1.18.4'
```

Comment:
do yo really have to add lombok as 4 dependencies?

Reply:
This is the way Gradle 5 takes case or annotation processing.